huxley/core/middlewares.py

```diff
@@ -1,24 +1,34 @@
 # Copyright (c) 2011-2015 Berkeley Model United Nations. All rights reserved.
 # Use of this source code is governed by a BSD License (see LICENSE).
 
-import json, logging
+import json, logging, sys, traceback
+
+from huxley.settings.main import DEBUG
 
 class ExceptionLoggerMiddleware(object):
-    def process_exception(self, request, exception):
-        logger = logging.getLogger('huxley')
-        logger.exception(exception)
 
-class ServerLoggingMiddleware(object):
     def process_exception(self, request, exception):
-        logger = logging.getLogger('huxley.server')
-        logger.exception(exception)
+        logger = logging.getLogger('huxley.exception')
+        exc_type, exc_value, exc_traceback = sys.exc_info()
+        exc_traceback = ''.join(traceback.format_exception(exc_type, exc_value, exc_traceback))
```

Comment:
Should we join on newline? Otherwise the stack trace would be hard to read as a big blob.